packages/engine/Source/DataSources/DataSourceDisplay.js

```diff
@@ -151,6 +175,7 @@ DataSourceDisplay.defaultVisualizersCallback = function (
       dataSource._primitives,
       dataSource._groundPrimitives
     ),
+    ...ExtraVisualizers.map((Visualizer) => new Visualizer(scene, entities)),
```

Comment:
Since `Visualizer` is already defined, I think this should become

```suggestion
    ...ExtraVisualizers.map((VisualizerClass) => new VisualizerClass(scene, entities)),
```

or something similarly descriptive.